src/com/kainos/tdp/employeemanager/Main.java

```diff
@@ -18,6 +18,12 @@ private static void runMenu() {
 			userInput = sc.nextInt();
 			validChoice = menu.checkUserChoiceValid(userInput);
 		} while(!validChoice);
+		
+		switch (userInput) {
+			case 1 : menu.addEmployee();
```

Comment:
```suggestion
			case 1:
				menu.addEmployee();
```
Align `break;` with this also